pkg/minikube/machine/client.go

```diff
@@ -147,8 +147,8 @@ func (api *LocalClient) Close() error {
 }
 
 // CommandRunner returns best available command runner for this host
-func CommandRunner(h *host.Host) (command.Runner, error) {
-	if h.DriverName == constants.DriverMock {
+func CommandRunner(h *host.Host, minikubeRunning bool) (command.Runner, error) {
```

Comment:
This API change doesn't feel right to me. The caller should first detect that the cluster is not running, and not ask for a runner in that case. Silently passing a mock back can result in unexpected behavior.

Reply:
What time of runner should be used?